index.html

```diff
@@ -150,5 +150,28 @@ <h2 class="list-of-skills">Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact-section">
+    <div class="contact-details">
+      <h2 class="paragraph_1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    </div>
+    <form class="form" action="https://formspree.io/f/{form_id}" method="post">
+      <ul>
+        <li><input type="text" placeholder="Full Name" maxlength="30" required></li>
```

Comment:
- [OPTIONAL] Kindly make sure the `input` has the `name` attribute which will help the [Formspree service](https://formspree.io/html) to extract the information from your form.